index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Anja Schmidt's Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="mobile_container">
+      <!-- Nav Bar -->
+      <nav>
+        <ul>
+          <li>Welcome</li>
```

Comment:
- [ ] If this is a logo, then it should be be a link(`<a>`), if you notice on most websites, the logo is usually a link that takes you to the home page. please kindly fix that.